ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/numberinput/v1/numberinput/clientlibs/site/js/numberinputview.js

```diff
@@ -87,7 +87,7 @@
         setModel(model) {
             super.setModel(model);
             // only initialize if patterns are set
-            if (this._model._jsonModel.editFormat || this._model._jsonModel.displayFormat
+            if (this._model._jsonModel.editFormat || this._model._jsonModel.displayFormat || this._model._jsonModel.displayValueExpression
```

Comment:
I'm expecting the displayValue property to return the value according to this expression that needs to happen at af-core level. So, af-core package needs an update. Without that, this PR seems incomplete. Am I missing something?

Reply:
Also, the json schema conformance tests are expected to break with these changes.

Reply:
@nit23uec yes af-core changes are yet to be merge. Once released I'll be updating the same in this PR.

Let me check the schema tests and confirm this